code/game/gamemodes/events/biomass.dm

```diff
@@ -202,10 +202,11 @@
 		var/area/Hallway = locate(type)
 
 		for(var/turf/simulated/floor/Floor in Hallway.contents)
-			if(Floor.contents.len <= 0)
+			if(Floor.contents.len <= 1)//There's always an /atom/movable/lighting_overlay on turfs

```

Comment:
Just saying, but not all turfs have a lighting overlay, only dynamic ones (you have to check for `lighting_use_dynamic` on the area and `dynamic_lighting` on the turf), and even then you shouldn't rely on that.